docs/en/platform-engineers/cue/patch-trait.md

```diff
@@ -97,15 +97,17 @@ By default, patch trait in KubeVela leverages the CUE `merge` operation. It has
 
 ### Strategy Patch
 
-The `strategy patch` is useful for patching array list.
+Strategy Patch is effective by adding annotation, and supports the following two ways
 
 > Note that this is not a standard CUE feature, KubeVela enhanced CUE in this case.
 
-With `//+patchKey=<key_name>` annotation, merging logic of two array lists will not follow the CUE behavior. Instead, it will treat the list as object and use a strategy merge approach:
+#### 1. With `+patchKey=<key_name>` annotation
+
+this is useful for patching array list, merging logic of two array lists will not follow the CUE behavior. Instead, it will treat the list as object and use a strategy merge approach:
```

Comment:
```suggestion
This is useful for patching array list, merging logic of two array lists will not follow the CUE behavior. Instead, it will treat the list as object and use a strategy merge approach:
```